### PR TITLE
Update mode.md

### PR DESCRIPTION
**Pull request by Legends**
Added possible values for mode at the beginning.
Replaced `provides` with `sets`, sounds better imo.

### CODE REVIEW

**Comment by Legends** [comment]
@montogeek

File: src/content/concepts/mode.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Providing the `mode` configuration option tells webpack to use its built-in opti
 
 `string`
 
-T> The default value for `mode` is `production`.
+T> Possible values for `mode` are: `development` or `production`(default).
```
Comment (montogeek) [inline src/content/concepts/mode.md]: What you think about adding `none`? It is a possible value as well.

**Comment by montogeek** [comment]
>Setting --mode to none is the same as omitting it?
If you omit it, it will use `production`

Sorry, I don't understand your second question.

**Comment by Legends** [comment]
Never mind, I just found the answer to my question, reading the docs helps in my case ;-)


> none --> Opts out of any default optimization options

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**